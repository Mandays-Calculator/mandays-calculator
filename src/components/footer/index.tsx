export { default as Footer } from "./Footer";
export { StyledFooterContainer, StyledFooterText } from "./styles";
