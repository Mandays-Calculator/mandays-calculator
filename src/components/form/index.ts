export * from "./checkbox";
export * from "./checkbox-group";
export * from "./text-field";
export * from "./date-picker";
