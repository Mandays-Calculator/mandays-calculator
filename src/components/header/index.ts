export {
  StyledAppBar,
  StyledToolBarContainer,
  StyledSearchSelect,
  popperStyle,
  StyledProjectTitle,
} from "./styles";
