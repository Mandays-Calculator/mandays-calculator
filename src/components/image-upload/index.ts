export { default as ImageUpload } from "./ImageUpload";
export * from "./styles";
