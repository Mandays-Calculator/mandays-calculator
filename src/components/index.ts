export * from "./form/text-field";
export * from "./form/Form";
export * from "./title";
