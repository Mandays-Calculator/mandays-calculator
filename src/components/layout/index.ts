export { default as Layout } from "./Layout";
export {
  StyledLayoutContainer,
  StyledLayoutContent,
  ContentContainer,
} from "./styles";
