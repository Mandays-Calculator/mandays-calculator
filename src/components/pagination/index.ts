export { default as Pagination } from "./Pagination";
export * from "./styles";
