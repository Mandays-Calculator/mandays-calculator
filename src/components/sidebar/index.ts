export {
  DrawerHeader,
  StyledItemText,
  StyledListItem,
  StyledListItemIcon,
  StyledDrawer,
  StyledList,
} from "./styles";

export { default as Sidebar } from "./Sidebar";
