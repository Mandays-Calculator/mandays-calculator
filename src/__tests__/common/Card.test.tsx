import { render } from "@testing-library/react";
import { Card } from "~/components";

describe("Card Component", () => {
  it("renders the component", () => {
    const { getByText, asFragment } = render(<Card title="Test">Card Content</Card>);

    expect(getByText("Test")).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
