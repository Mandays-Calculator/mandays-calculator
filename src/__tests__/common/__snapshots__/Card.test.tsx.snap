// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component renders the component 1`] = `
<DocumentFragment>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <div
      class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
    >
      <h6
        class="MuiTypography-root MuiTypography-h6 css-tjvwag-MuiTypography-root"
      >
        Test
      </h6>
      Card Content
    </div>
  </div>
</DocumentFragment>
`;
